### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by syedmdsaifihassan**
Dynamic Timeline using vanilla js

## Issue #416 
  - https://github.com/Vishal-raj-1/Awesome-JavaScript-Projects/issues/416

### Describe the changes you've made
I have created a Dynamic Timeline using Vanilla js 

### Checklist:
<!--
Example how to mark a checkbox:-
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
timeline

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 let projectData = [
   {
+<<<<<<< Updated upstream
```
Comment (991rajat) [inline script.js:3]: Merge Conflict is there, Resolve them please.
Only add your project only.

**Review by 991rajat — CHANGES_REQUESTED** [review]
Fix the conflict, 
There are lot of projects which is not required, it'll create duplicacy when merging,
Only add your project.

File: script.js (hunk 1 above)
Comment (syedmdsaifihassan) [reply script.js]: Hi @991rajat I have resolved the conflict Please review again.

**Comment by 991rajat** [comment]
@syedmdsaifihassan You created a new PR but there is no code for html/css of the timeline app you created. 
This PR can't be merged as it is removing so many projects of other participants,
Just add the code in the new PR and push also don't remove the project from new PR as well.

I'm closing this one as you created new PR.

**Pull request closed without merging**